src/lib/init.ts

```diff
@@ -58,19 +56,94 @@ import {
   getREADMEPath,
 } from "../lib/paths.js";
 import { confirm, input, list } from "../lib/prompt.js";
+import CLIPackageJSON from "../versions/cli.package.json" assert { type: "json" };
 import versions from "../versions.json" assert { type: "json" };
 
 import { addService } from "./addService.js";
 import { commandObj, isInteractive } from "./commandObj.js";
+import { setEnvConfig } from "./configs/globalConfigs.js";
+import { initNewEnvConfig } from "./configs/project/env.js";
 import { initNewReadonlyServiceConfig } from "./configs/project/service.js";
 import { initNewWorkersConfig } from "./configs/project/workers.js";
+import type { ContractsENV } from "./const.js";
 import { addCountlyEvent } from "./countly.js";
 import { generateNewModule } from "./generateNewModule.js";
 import { ensureAquaImports } from "./helpers/aquaImports.js";
-import { jsonStringify } from "./helpers/jsonStringify.js";
+import { jsonStringify } from "./helpers/utils.js";
 import { initMarineCli } from "./marineCli.js";
+import {
+  ensureCustomRelays,
+  resolveFluenceEnv,
+  local,
+  multiaddrsToNodes,
+} from "./multiaddres.js";
 import { ensureSrcAquaMainPath } from "./paths.js";
 
+const DISABLE_TS_AND_ES_LINT = `/* eslint-disable */
+// @ts-nocheck`;
+
+const NODES_CONST = "nodes";
+
+const getPeersImportStatement = (peersToImport: string): string => {
+  return `import { ${peersToImport} as ${NODES_CONST} } from "@fluencelabs/fluence-network-environment";`;
+};
+
+const getCustomPeers = (peers: Node[]) => {
+  return `const ${NODES_CONST} = ${jsonStringify(peers)}`;
+};
+
+const getPeersImportInJS = (
+  fluenceEnvOrCustomRelays: ContractsENV | Array<string>,
+) => {
+  if (Array.isArray(fluenceEnvOrCustomRelays)) {
+    return getCustomPeers(multiaddrsToNodes(fluenceEnvOrCustomRelays));
+  }
+
+  return {
+    kras: getPeersImportStatement("krasnodar"),
+    stage: getPeersImportStatement("stage"),
+    testnet: getPeersImportStatement("testNet"),
+    local: getCustomPeers(local),
+  }[fluenceEnvOrCustomRelays];
+};
+
+export const getJsTemplateIndexJsContent = (
+  fluenceEnvOrCustomRelays: ContractsENV | Array<string>,
+) => {
+  return `${DISABLE_TS_AND_ES_LINT}
+import { Fluence } from "@fluencelabs/js-client";
+${getPeersImportInJS(fluenceEnvOrCustomRelays)}
+
+import {
+  helloWorld,
+  helloWorldRemote,
+  getInfo,
+  getInfos,
+} from "./aqua/main.js";
+
+const peerIds = ${NODES_CONST}.map(({ peerId }) => peerId);
+const connectTo = ${NODES_CONST}[0].multiaddr;
+if (typeof connectTo !== "string") {
+  throw new Error("connectTo is not a string");
+}
+
+const main = async () => {
+  await Fluence.connect(connectTo);
+
+  const helloWorldResult = await helloWorld("Fluence");
+  const helloWorldRemoteResult = await helloWorldRemote("Fluence");
+  const getInfoResult = await getInfo();
+
+  console.log(helloWorldResult);
+
+  process.exit(0);
```

Comment:
why not `await Fluence.disconnect()` ?

Reply:
for this example it prefectly works without it. But wouldn't harm. Will add

Reply:
I tried refactoring it a bit - check it out after I push